package prueba.prueba.exception;

public class GlobalExceptionHandler {
}
